src/sentry/integrations/opsgenie/integration.py

```diff
@@ -64,24 +63,29 @@
 class InstallationForm(forms.Form):
     base_url = forms.ChoiceField(
         label=_("Base URL"),
-        # help_text=_("Either https://api.opsgenie.com/ or https://api.eu.opsgenie.com/"),
-        # widget=forms.TextInput(attrs={"placeholder": "https://api.opsgenie.com/"}),
         choices=[
             ("https://api.opsgenie.com/", "api.opsgenie.com"),
             ("https://api.eu.opsgenie.com/", "api.eu.opsgenie.com"),
         ],
     )
+    provider = forms.CharField(
+        label=_("Account Name"),
+        help_text=_("Example: 'hello' for https://hello.app.opsgenie.com/"),
```

Comment:
optional nit: I don't think we have a standard text for stuff like this but I suggest using `acme`, or `example` or something like `my-company`